.eslintrc.json

```diff
@@ -56,6 +58,22 @@
     ],
     "quotes": ["error", "single", { "allowTemplateLiterals": true, "avoidEscape": true }]
   },
+  "settings": {
+    "import/core-modules": 
+      [
+        "googlesitekit-data",
+        "googlesitekit-modules",
+        "googlesitekit-api",
+        "googlesitekit-widgets",
+        "@wordpress/i18n__non-shim",
+        "@wordpress/api-fetch__non-shim"
+      ],
+      "import/resolver": {
+        "alias": [
+          ["tests/js/utils", "./tests/js/utils"]
```

Comment:
I was researching this for a while. I had to install another package `eslint-import-resolver-alias` and then add this config

This was the most popular way to do this online :smile:

Reply:
This should be replaced with a normal relative import; it's an old import that we used in the past before we moved everything (except above) to relative imports.

Reply:
Sounds great @aaemnnosttv 

Aliases are a great idea in principle but they cause problems in storybook and testing and etc :smile:

Reply:
This rule is removed and all failing imports have been changed to relative @aaemnnosttv  :+1: